json-schemas/interface-methods/protocol-rule-set.json

```diff
@@ -106,10 +106,54 @@
           "minimum": 0
         }
       }
+    },
+    "$tags": {
+      "type": "object",
+      "minProperties": 1,
+      "patternProperties": {
+        "requiredTags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "allowUndefinedTags": {
+          "type": "boolean"
+        },
+        "^(?!requiredTags$|allowUndefinedTags$).*$": {
```

Comment:
Should we prefix `requiredTags` and `allowUndefinedTags` with `$` for consistency with patterns we've established?

Assuming we do, this regular expression can be simplified.

Reply:
Yeah I'll do that, that was actually in my initial design, don't really remember why i removed it after reviewing a proposal with Daniel, but I prefer it as well.